docs/components/logos/TurboAnimated.tsx

```diff
@@ -0,0 +1,126 @@
+import { AnimatePresence, motion, Variants } from "framer-motion";
+import { useRouter } from "next/router";
+
+type LogoProps = {
+  className?: string;
+  height?: number;
+};
+
+const TurboAnimated = ({ height = 32, className = "" }: LogoProps) => {
+  const router = useRouter();
+  const showTurboRepoLogo = !router.route.startsWith("/pack");
```

Comment:
You can `useTurboSite` (from [here](https://github.com/vercel/turbo/blob/main/docs/components/SiteSwitcher.tsx#L5))